### PR TITLE
Revised TypeScript declaration

### PR DESCRIPTION
**Pull request by JarryShaw**
* added `$.term.syntax` function (provided by prism)
 * added `height` option to Terminal instance (as it should be)
 * fixed several typos
 * lint the file

### CODE REVIEW

File: js/jquery.terminal.d.ts [hunk 1]
```diff
@@ -358,9 +359,10 @@ type CmdOptions = {
...
     mobileDelete?: boolean;
 }
-type CmdOption = "mask" | "caseSensitiveSearch" | "historySize" |  "prompt" | "enabled" |
-    "history" | "tabs" | "onPositionChange" |  "clickTimeout" |  "holdTimeout" | "onPaste" |
-    "holdRepeatTimeout" |  "repeatTimeoutKeys" | "width" | "historyFilter" | "commands" |
+
+type CmdOption = "mask" | "caseSensitiveSearch" | "historySize" | "prompt" | "enabled" |
+    "history" | "tabs" | "onPositionChange" | "clickTimeout" | "holdTimeout" | "onPaste" |
+    "holdRepeatTimeout" | "repeatTimeoutKeys" | "width" | "historyFilter" | "commands" |
     "char_width" | "onCommandChange" | "name" | "keypress" | "keydown" | "mobileDelete";
```
Comment (jcubic) [inline js/jquery.terminal.d.ts]: What did you change here?

**Comment by jcubic** [comment]
Beside this question, it looks good. Thanks for the fix.

File: js/jquery.terminal.d.ts (hunk 1 above)
Comment (JarryShaw) [reply js/jquery.terminal.d.ts]: Linting. There’re some `|` operators having two spaces before and/or after.

**Comment by jcubic** [comment]
I really need to fix the broken test ASAP, (there is one typescript check that validate d.ts file, it don't run because of previous errors) probably will do at the weekend.

**Pull request merged**